src/js/components/Accordion/Accordion.js

```diff
@@ -82,7 +78,7 @@ class Accordion extends Component {
       <Box role='tablist' {...rest} overflow='auto'>
         {Children.toArray(children).map((panel, index) => (
           <AccordionContext.Provider
-            key={`accordion-panel_${index}`}
+            key={`accordion-panel_${index + 0}`}
```

Comment:
why is this needed?

Reply:
It avoids eslint complaints about using index as a key.
Would you prefer disabling the eslint rule? I actually wonder if an inline /* eslint-disable */ would be even better, to make it clearer. I didn't want to disable the eslint rule globally because sometimes there is a way to use a non-index key, and we should do that wherever possible.